src/backer/cardaning.py

```diff
@@ -71,10 +71,7 @@ def __init__(self, name='backer', hab=None, ks=None):
     def publishEvent(self, event: bytearray):
         # TODO: Change to the arrays
         self.pending_kel = event
```

Comment:
This will publish an event 1 at a time, each in a separate transaction. What I asked for is if you can publish all unpublished KELs in one go in one transaction?

Reply:
hmm
it will combine all the evenst into one, it should be arrays|bytearrays
The thing is AuxiliaryData(Metadata({1: value})) is not accept the bytesaray
https://github.com/cardano-foundation/cardano-backer/blob/feat/watchpoc/src/backer/cardaning.py#L95

Reply:
You just need to concatenate all of the byte arrays BEFORE splitting it into the 64 byte chunks: https://github.com/cardano-foundation/cardano-backer/blob/feat/watchpoc/src/backer/cardaning.py#L92

Reply:
okay, will update it